onnx/common/version.h

```diff
@@ -8,6 +8,6 @@
 namespace ONNX_NAMESPACE {
 
 // Represents the most recent release version. Updated with every release.
-constexpr const char* LAST_RELEASE_VERSION = "1.9.0";
+constexpr const char* LAST_RELEASE_VERSION = "1.9.100";
```

Comment:
this should be 1.10.0

Reply:
Switched it back to 1.10 based on comments.